src/main/AuthenticationResponse.ts

```diff
@@ -288,14 +289,57 @@ async function createSIOPResponsePayload(
     iat: Date.now() / 1000,
     exp: Date.now() / 1000 + (resOpts.expiresIn || 600),
     registration,
-    vp_token,
-    verifiable_presentations,
   };
+  if (resOpts.vp && resOpts.vp[0].location === PresentationLocation.ID_TOKEN) {
+    const id_token = {
+      iss: ResponseIss.SELF_ISSUED_V2,
+      aud: verifiedJwt.payload.redirect_uri,
+      iat: Date.now() / 1000,
+      exp: Date.now() / 1000 + (resOpts.expiresIn || 600),
+      sub: resOpts.did,
+      //TODO: We don't have this value in the payload
+      auth_time: verifiedJwt.payload.auth_time,
+      nonce: nonce,
+      _vp_token: {
+        // right now pex doesn't support presentation_submission for multiple VPs (as well as multiple VPs)
```

Comment:
Can we create a ticket that we refine later in Jira and link to it

Reply:
created 3:
1. to support `presentation_submission` outside of VP in PEX
2. to support multiple VPs in PEX
3. to handle multiple VPs in this project (did-auth-siop)